src/linear_algebra/direct_sum_module.lean

```diff
@@ -22,9 +22,11 @@ variables {R ι M}
 instance : semimodule R (direct_sum ι M) := dfinsupp.to_semimodule
 
 variables R ι M
-def lmk : Π s : finset ι, (Π i : (↑s : set ι), M i.1) →ₗ[R] direct_sum ι M :=
+/-- This is like `mk`, but indexed by a finset. -/
+def lmk : Π s : finset ι, (Π i : (↑s : set ι), M i.val) →ₗ[R] direct_sum ι M :=
 dfinsupp.lmk M R
 
+/-- The inclusion from the product type to the direct sum, as a linear map. -/
 def lof : Π i : ι, M i →ₗ[R] direct_sum ι M :=
 dfinsupp.lsingle M R
```

Comment:
I don't think this is the correct doc-string! Note that there isn't a map from the product to the sum at all. This would be a lot clearer if the statement were 
```
def lof (i : ι) : M i →ₗ[R] direct_sum ι M :=
```

Reply:
Fixing it.